
This should fix the compilation for new kernels.

  - alan.rock@glaramara.freeserve.co.uk

diff -Naur iproute2.rockpatched/Makefile iproute2/Makefile
--- iproute2.rockpatched/Makefile	2003-08-27 13:53:08.000000000 +0100
+++ iproute2/Makefile	2003-08-27 13:53:35.000000000 +0100
@@ -20,11 +20,11 @@
 
 ifeq ($(LIBC_INCLUDE)/socketbits.h,$(wildcard $(LIBC_INCLUDE)/socketbits.h))
   ifeq ($(LIBC_INCLUDE)/net/if_packet.h,$(wildcard $(LIBC_INCLUDE)/net/if_packet.h))
-    GLIBCFIX=-I../include-glibc -include ../include-glibc/glibc-bugs.h
+    GLIBCFIX=-include ../include-glibc/glibc-bugs.h
   endif
 endif
 ifeq ($(LIBC_INCLUDE)/bits/socket.h,$(wildcard $(LIBC_INCLUDE)/bits/socket.h))
-  GLIBCFIX=-I../include-glibc -include ../include-glibc/glibc-bugs.h
+  GLIBCFIX=-include ../include-glibc/glibc-bugs.h
 endif
 
 CC = gcc
